src/main/resources/com/google/api/codegen/nodejs/version_index.snip

```diff
@@ -93,17 +96,13 @@
 @end
 
 @private messages(isGcloud, version, stub)
-    extend({@version}Protos, gaxGrpc.load([{
-      @if isGcloud
-        root: require('google-proto-files')('..'),
-      @else
-        # This does not follow the import in main.snip since version index is not generated
-        # for unversioned Apis. This will be changed when top level indexes are supported
-        # for unversioned Apis.
-        root: require('path').join(__dirname, '..', '..', 'proto'),
-      @end
-      file: '{@stub.protoFileName}'
-    }]).{@stub.namespace});
+    extend({@version}Protos, gaxGrpc.loadObject(
+      # This does not follow the import in main.snip since version index is not generated
```

Comment:
I do not understand what this comment means. (But that might be okay?)

Reply:
This was an existing comment. Essentially says that the main.snip has some logic to determine how far back in the directory the proto files are. This file will always be located 1 directory away from the proto directory.